pimcore/config/helper.php

```diff
@@ -126,53 +126,54 @@ function object2array($node)
 
 /**
  * @param  $args
- * @return bool|string
+ * @return false|string
  */
 function array_urlencode($args)
 {
     if (!is_array($args)) {
         return false;
     }
-    $out = '';
-    foreach ($args as $name => $value) {
-        if (is_array($value)) {
-            foreach ($value as $key => $val) {
-                $out .= urlencode($name).'['.urlencode($key).']'.'=';
-                $out .= urlencode($val).'&';
-            }
-        } else {
-            $out .= urlencode($name).'=';
-            $out .= urlencode($value).'&';
-        }
-    }
-
-    return substr($out, 0, -1); //trim the last & }
+    
+    return str_replace(['%5B', '%5D'], ['[', ']'], http_build_query($args));
```

Comment:
Good question :) 
If it doesn't matter, I'd also prefer removing the `str_replace`. 

Thanks!